client/my-sites/plugins/education-footer/use-lists.tsx

```diff
@@ -0,0 +1,56 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { useTranslate } from 'i18n-calypso';
+
+export const useEducationLinksList = () => {
+	const translate = useTranslate();
+
+	return [
+		{
+			id: 'website_building',
+			title: translate( 'What Are WordPress Plugins and Themes? (A Beginner’s Guide)' ),
+			url: localizeUrl(
+				'https://wordpress.com/go/website-building/what-are-wordpress-plugins-and-themes-a-beginners-guide/'
+			),
+		},
+		{
+			id: 'customization',
+			title: translate( "How to Use WordPress Plugins: The Complete Beginner's Guide" ),
+			url: localizeUrl( 'https://wordpress.com/go/website-building/how-to-use-wordpress-plugins/' ),
```

Comment:
This is the only noticeable change. The old link for this card didn't/doesn't work, and I couldn't find anything to suggest that the article even existed.

Reply:
Thanks for reporting this, this was also handled and merged here: https://github.com/Automattic/wp-calypso/pull/87722

And that is probably the reason why this PR has a conflict that needs to be solved before merging it :|